src/lib/reasoners/bitv.ml

```diff
@@ -104,7 +117,7 @@ module Shostak(X : ALIEN) = struct
 
   let compare_simple_term = compare_alpha_term (fun st1 st2 ->
       match st1, st2 with
-      | Cte b,Cte b' -> compare b b'
+      | Cte true, Cte true | Cte false, Cte false -> 0
```

Comment:
We should use:

```suggestion
      | Cte b, Cte b' -> Bool.compare b b'
```

Reply:
Is it faster or that is a matter of taste? I wrote `Bool.compare` also but since Steven used the other style, I thought it could be faster.

Reply:
The performance difference should be negligible (`Bool.compare` is most probably inlined anyways), but the version with `Bool.compare` has less mental overhead: since it is a catch-all case, one doesn't have to keep track of the cases `true, false` and `false, true` while reading the rest of the code, and it is more obviously correct.